bin/update-dependencies

```diff
@@ -8,7 +8,7 @@ if [ -z "$2" ] ; then
   echo "gem version is required!" && exit 2;
 fi
 
-for gemspec in $(git ls-files \*.gemspec)
+for gemspec in $(git ls-files \*Gemfile)
```

Comment:
I know we usually use this for dev dependencies, but prior to this PR, the script could also be used with general dependencies too. if this is all that needs to be changed, what do you think about updating documentation (possibly in `CONTRIBUTING.md`) to note how to adjust the script to update dependencies in gemspecs?

Reply:
True! I'll make sure this supports both.